docs/docusaurus/docs/core/expectation_suites/manage_expectation_suites.md

```diff
@@ -0,0 +1,294 @@
+---
+sidebar_label: 'Manage Expectation Suites'
+title: 'Manage Expectation Suites'
+description: Create and manage sets of Expectations in Python with GX Core.
+---
+
+An Expectation Suite contains a group of Expectations that describe the same set of data.  All of the Expectations that you apply to your data will grouped into Expectation Suites.
+
+## Prerequisites
+
+This guide assumes you have:
+
+- Installed Python.
+- Installed the GX Core library.
+
+## Create an Expectation Suite
+
+1. Import the GX Core library and the `ExpectationSuite` class.
```

Comment:
```suggestion
1. Import the GX Core library and the `ExpectationSuite` class:
```
You'll need to decide if you want to introduce code samples with colons. If you do, add them where necessary. Also, do you want to add _run the following code_ to your statements?